CHANGES.md

```diff
@@ -42,6 +42,7 @@ replace with next unreleased version
 
 ### NEW FEATURES
 
+- [DDF-2467](https://codice.atlassian.net/browse/DDF-2467) Initial support for Solr Cloud
```

Comment:
Long term I would like to see us move towards an approach like Chrome's experimental flags.